### ISSUE TITLE
[BUG] Home screen widgets not loading since v0.3.23

### ISSUE THREAD

**Issue by joeljwilliams** [jurialmunkey/skin.arctic.fuse#212]
### Skin section

Home Widgets

### Current Behavior

After the update to v0.3.23 my home screen widgets all load up blank. Tried rerunning wizard and selecting different views but all result in the same empty widgets.

### Expected Behavior

Spotlights and widgets to load with content.

### Steps To Reproduce

1. Launch Kodi
2. Browse through home screen widgets and spotlights

### Screenshots and Additional Info

Device: Shield Pro 2019
Kodi version: 21.0-BETA1 (20.90.801)
Skin version: v0.3.25

[kodi.log](https://github.com/jurialmunkey/skin.arctic.fuse/files/13334679/kodi.log)

![image](https://github.com/jurialmunkey/skin.arctic.fuse/assets/102667/288924b3-b927-4bb8-9aa7-060371373c39)
![image](https://github.com/jurialmunkey/skin.arctic.fuse/assets/102667/97aae095-8561-47e5-b1c9-01a45334b4cb)
![image](https://github.com/jurialmunkey/skin.arctic.fuse/assets/102667/f8877a6a-4b55-4cd3-b45e-a7672174bc28)

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
Looks like you are using the widget set for the library but dont have any library content... hence empty widgets.

You'll need to add widgets that aren't library based. There's prebuilt configurations from the "prebuilt" button on the side of the configuration dialog.

**Comment by joeljwilliams** [comment]
Yeah I was using the default TMdb helper ones. Thanks I'll have a look at the prebuilts.
